test/ptc/e2e/system_test.clj

```diff
@@ -49,23 +49,20 @@
         cloud-prefix     (jms/cloud-prefix bucket workflow)]
     (jms-tools/queue-messages 1 environment message)
     (testing "Files are uploaded to the input bucket"
-      (let [params (str cloud-prefix "/params.txt")
-            ptc    (str cloud-prefix "/ptc.json")
-            _      (timeout 360000 #(gcs/wait-for-files-in-bucket cloud-prefix [ptc]))
-            ; Dodge rarely observed race condition where `cat` errors even though `ls` shows the file
-            _      (.sleep TimeUnit/SECONDS 1)
-            {:keys [notifications]} (gcs/gcs-edn ptc)
-            pushed (utils/pushed-files (first notifications) params)
-            gcs    (timeout 180000 #(gcs/wait-for-files-in-bucket cloud-prefix pushed))]
-        (is (not= gcs ::timed-out) "Timeout waiting for files to upload")
-        (let [diff (set/difference (set gcs) (set pushed))]
-          (is (= diff (set [ptc])) "Files in bucket do not match expected files")
+      (let [params   (str cloud-prefix "/params.txt")
+            ptc      (str cloud-prefix "/ptc.json")
+            ptc-json (timeout 360000 #(gcs/wait-for-files-in-bucket [ptc]))]
+        (is (not= ptc-json ::timed-out) "Timed out waiting for ptc.json to upload")
+        (let [{:keys [notifications]} (gcs/gcs-edn ptc)
+              pushed                  (utils/pushed-files (first notifications) params)
+              pushes-files            (timeout 180000 #(gcs/wait-for-files-in-bucket pushed))]
```

Comment:
Minor suggestion: How about changing "pushed" to "expected-files", and "pushes-files" to "pushed-files"?

Reply:
Sure!